transcriptic/analysis/kinetics.py

```diff
@@ -49,7 +49,8 @@ def __init__(self, datasets):
             raise RuntimeError("Input Datasets must all be of the same type.")
         self.operation = operation_set.pop()
         if self.operation not in ["absorbance", "fluorescence", "luminescence"]:
-            raise RuntimeError("%s has to be of type absorbance, fluorescence or luminescence" % self.operation)
+            raise RuntimeError(f"{self.operation} has to be of type absorbance, "
+                               f"fluorescence or luminescence")
```

Comment:
is the f-string required here?
`f"{self.operation} has to be of type absorbance, fluorescence or luminescence"`

Reply:
hmm not really sure what you mean here by if its required...
Do you mean if we should be using f-string interpolation for this case? If so, I'm not sure what's different about this vs. other cases.